openstack/clustering/v1/profiles/results.go

```diff
@@ -29,9 +29,18 @@ func (r commonResult) Extract() (*Profile, error) {
 	return s.Profile, err
 }
 
+// ExtractProfiles provides access to the list of profiles in a page from the List operation.
+func ExtractProfiles(r pagination.Page) ([]Profile, error) {
+	var s struct {
+		Profiles []Profile `json:"profiles"`
+	}
+	err := (r.(ProfilePage)).ExtractInto(&s)
+	return s.Profiles, err
+}
+
 type Spec struct {
 	Type       string                 `json:"type"`
-	Version    string                 `json:"version"`
+	Version    interface{}            `json:"version"`
```

Comment:
I think this should be left as a string. If `Version` requires some smoothing over, then it should go in the `UnmarshalJSON` method similar to here:

https://github.com/gophercloud/gophercloud/blob/master/openstack/clustering/v1/policies/results.go

Reply:
Not sure whats the best way to handle this one. Might talk to upstream about change it to a string. For policies, json is returning as "1.0" but profiles, json is returning as 1.0. Also when golang converts from float to string it becomes "1". Also hopefully profile spec version doesn't allow "1.0.1" in the future.

Reply:
Created a bug for senlin to have json always returns string. But might be good idea for older senlin versions.